buildSrc/build.gradle

```diff
@@ -13,6 +13,7 @@ apply from: rootProject.file('../dependencies.gradle')
 
 repositories {
   mavenCentral()
```

Comment:
I prefer only mavenCentral. Jcenter is only temporary until Google resumes publishing the plugin there.